apps/configurator/lib/configurator_web/controllers/game_configuration_html/game_configuration_form.html.heex

```diff
@@ -2,6 +2,12 @@
   <.error :if={@changeset.action}>
     Oops, something went wrong! Please check the errors below.
   </.error>
+  <.input
```

Comment:
I think I can move this input to a single and centralized place but to keep it simple I just copy pasted it